eva_sub_cli/semantic_metadata.py

```diff
@@ -121,8 +122,13 @@ def check_all_analysis_run_accessions(self):
     def check_accession_in_ena(self, ena_accession, accession_type, json_path):
         try:
             res = download_xml_from_ena(f'https://www.ebi.ac.uk/ena/browser/api/xml/{ena_accession}')
-        except Exception:
-            self.add_error(json_path, f'{accession_type} {ena_accession} does not exist in ENA or is private')
+        except HTTPError as e:
+            if e.response.status_code == 500:
+                self.add_error(json_path, f'{accession_type} {ena_accession} could not be resolve on ENA because the service is unavailable. try again later.')
+            else:
+                self.add_error(json_path, f'{accession_type} {ena_accession} does not exist in ENA or is private')
+        except Exception as e:
+            self.add_error(json_path, f'Unexpected error occurred when checking {accession_type} {ena_accession}')
```

Comment:
This is to differentiate between private samples and failed API calls in the report